src/version.ts

```diff
@@ -0,0 +1 @@
+export default '0.0.11'
```

Comment:
What if we read the version directly from package.json? Is it okay to have 2 sources of the value synchronised automatically?

Reply:
Initially, I wanted to read the package.json to keep it simple. However, to do that, you need to read that file (`const packageJson = require('./package.json')` for example) , and this library is intended to be used within other apps with their own package.json file; I prefer to avoid possible bugs due to the resolution of the relative path.